<launch>
  <arg name="bag_path"/>
  <arg name="output_path"/>
  <node pkg="rosbag" type="play" name="rosbag" required="true" args="$(arg bag_path)"/>
  <node name="extract" pkg="image_view" type="image_saver" respawn="false"
    required="true" output="screen" cwd="ROS_HOME">
    <param name="filename_format" type="string" value="$(arg output_path)/frame-%06d.color.%s"/>
    <remap from="image" to="/usb_cam/image_raw"/>
  </node>
</launch>
